### DISCUSSION TITLE
Not able to upload screenshots using sorry-cypress chart with minio

### DISCUSSION THREAD

**Discussion by dawogid** [sorry-cypress/sorry-cypress#827] [Q&A]
Hi, I'm not able to upload and see screenshots in sorry-cypress dashboard. I'm using sorry-cypress helm chart with the below values.yaml:

```
director.screenshotsDriver: "../screenshots/minio.driver"
director.environmentVariables.executionDriver: "../execution/mongo/driver"

minio.enabled: true
minio.service.port: 80
minio.endpoint: sorry-cypress-minio.mydomain.net
minio.url: https://sorry-cypress-minio.mydomain.net
minio.readUrlPrefix: ""
minio.defaultBucket.enabled: true
minio.defaultBucket.name: sorry-cypress
minio.defaultBucket.policy: public
minio.persistence.size: 10Gi
```
I don't want to use AWS s3, so I left it's configuration with default values in values.yaml:

```
s3.bucketName: example-bucket
s3.region: us-east-1
s3.accessKeyId: ""
s3.secretAccessKey: ""
s3.acl: "public-read"
s3.readUrlPrefix: ""
s3.videoKeyPrefix: ""
s3.imageKeyPrefix: ""
```

I created an ingress configuration that points from "sorry-cypress-minio.mydomain.net" on port 80 to the "sorry-cypress-minio" service.

With this setup, I am successfully running Cypress tests using the sorry-cypress director. The test results are visible on the sorry-cypress dashboard.

Furthermore, I can log in to Minio under https://sorry-cypress-minio.mydomain.net without any issues, bucket sorry-cypress is created. The sorry-cypress-director pod starts up smoothly, showing no errors during its initialization.

However, there is a matter of concern that has come to my attention, which relates to an error appearing in the director's logs during test execution:

`"error":{"type":"S3Error","message":"","stack":"S3Error: \n    at Object.parseError (/app/node_modules/minio/dist/main/xml-parsers.js:71:11)\n    at /app/node_modules/minio/dist/main/transformers.js:166:22\n    at DestroyableTransform._flush (/app/node_modules/minio/dist/main/transformers.js:90:10)\n    at DestroyableTransform.prefinish (/app/node_modules/readable-stream/lib/_stream_transform.js:129:10)\n    at DestroyableTransform.emit (node:events:513:28)\n    at prefinish (/app/node_modules/readable-stream/lib/_stream_writable.js:611:14)\n    at finishMaybe (/app/node_modules/readable-stream/lib/_stream_writable.js:620:5)\n    at endWritable (/app/node_modules/readable-stream/lib/_stream_writable.js:643:3)\n    at DestroyableTransform.Writable.end (/app/node_modules/readable-stream/lib/_stream_writable.js:571:22)\n    at IncomingMessage.onend (node:internal/streams/readable:693:10)","amzRequestid":null,"amzId2":null,"amzBucketRegion":null},"instanceId":"4e419abb-af42-4c9d-8306-f47baf94fe64","msg":"Unable to get upload instructions"}`

Could there be something that I overlooked in the configuration?

Currently I'm using:

- cypress version: 12.17.1
- sorry-cypress helm chart version: 2.5.3

**Comment by k0ezeN** [comment]
hello did you find the solution, have the exact same error
